clients/c/src/orpClient.c

```diff
@@ -292,6 +292,32 @@ static le_result_t orp_ClientMessageSend
 }
 
 
+//--------------------------------------------------------------------------------------------------
+/**
+ * Respond to a sync packet
+ */
+//--------------------------------------------------------------------------------------------------
+le_result_t orp_Respond
+(
+    enum orp_PacketType type,
+    int status
+)
+{
+    struct orp_Message message;
+
+    switch (type)
+    {
+        case ORP_RESP_HANDLER_CALL: break;
+        case ORP_RESP_SENSOR_CALL: break;
+        case ORP_SYNC_SYNACK: break;
+        case ORP_SYNC_ACK: break;
+        default: return LE_BAD_PARAMETER;
```

Comment:
I would prefer to have the break statement on a separate line. Moreover it can be simplified
```
        case ORP_RESP_HANDLER_CALL:
        case ORP_RESP_SENSOR_CALL:
        case ORP_SYNC_SYNACK:
        case ORP_SYNC_ACK: 
              // Supported response message types 
              break;
        default: 
              return LE_BAD_PARAMETER;
```